.github/workflows/npm-audit.yml

```diff
@@ -15,6 +15,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Setup NodeJS
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
```

Comment:
Let's use `BUILD_NODE_VERSION` variable instead of hardcoded value.

Reply:
Is that a global variable?

Reply:
@yashrajk17 yes, I just added it. You can use it as `{{ vars.BUILD_NODE_VERSION }}`